src/css/application.css

```diff
@@ -473,6 +473,20 @@ body {
   cursor: pointer;
 }
 
+.errorList-error-message {
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+
+  -ms-word-break: break-all;
+  word-break: break-all;
+  word-break: break-word;
```

Comment:
I’m not an expert, but I don’t think this is necessary—I think all the [browser versions Popcode supports](https://github.com/popcodeorg/popcode/blob/master/config/browsers.json) can deal with `break-all` just fine?